Add a blurp about i18n fonts (#331)
pxblue/doc-it: src/docs/style/typography.mdx

<React.Fragment>
    <ImageGrid
        images={[ApplicationImg]}
        regularWidth
        caption={['1: Card layout using default PX Blue styles', '2: Card layout using custom created styles']}
    />
    <ImageGrid
        images={[HeadlineOptionsImg]}
        regularWidth
        caption={[
            '1: Two line headline using default PX Blue styles',
            '2: Two line headline using custom created styles',
        ]}
    />
    <Divider />
</React.Fragment>

# Internationalization

<ImageGrid
    images={[SimplifiedChineseImg]}
    regularWidth
    caption={['Use "Noto Sans CJK SC" font for Simplified Chinese.']}
/>

> PX Blue recommends using [Noto Sans](https://www.google.com/get/noto/) for all non-western languages. You should choose the appropriate font based on the language you are designging for.

Reviewer: ```suggestion
For non-western languages, we recommend using [Noto Sans](https://www.google.com/get/noto/). You should select the appropriate font variation based on the language(s) you are designing for.
```
